File: src/main/java/com/fly/Box3.java
package com.fly;

public class Box3 extends Box {
    //設定箱子3長寬高名字
    public Box3() {
        super(23, 14, 13, "Box3");
    }

    @Override
    public boolean validate(float length, float width, int height) {
        if (length <= this.length && width <= this.width && height <= this.height) {
            return true;
        } else {
            return false;
        }
    }

    @Override
    public void getName() {
        System.out.println(this.name);
    }
}

File: src/main/kotlin/com/fly/Box3.kt
package com.fly.kotlin


class Box3 : Box(23f, 14f, 13, "Box3") {

    override fun validate(length: Float, width: Float, height: Int): Boolean {
        if (length < this.length && width <= this.width && height <= this.height) {
            return true
        }
        return false
    }

    override fun getName() = print(this.name)


}